content/fr/articles/js/angular/components/index.md

```diff
@@ -0,0 +1,166 @@
+---
+date: "2016-01-24"  
+title: "Angular 1.5 : un pas de plus vers les web-components"  
+tags:
+  - javascript
+  - angular
+  - web-component  
+authors:
+  - Freezystem
+---
+
+Ce post est écrit à la lumière du 
+[changelog d'Angular 1.5.0 rc1](https://github.com/angular/angular.js/blob/master/CHANGELOG.md#150-rc1-quantum-fermentation-2016-01-15).  
+Toutes information est succeptible d'évoluer au cours des publications de 
+nouvelles versions du framework.
+
+## Le petit nouveau : `.component()`
+
+Avec l'arrivée imminente de la version 2, Angular commence lentement à préparer la
+transition et apporte de nombreux changements à la v1 pour tenter de 
+combler l'écart entre les deux et rendre les changements moins pénibles.  
+
+On voit donc progressivement apparaitre de nouvelles fonctionnalités comme :
+ - ~~un [nouveau router](https://angular.github.io/router/) ?~~ 
+ - la possibilité d'[annuler une resource](https://docs.angularjs.org/api/ngResource/service/$resource#cancelling-requests)
+ - les transclusions multiples
+ - la compilation paresseuse des transclusions
+
+Mais le point qui nous intéresse particulièrement ici n'est autre que 
+le nouveau _helper_ permettant la déclaration de similis 
+(composants web)[https://fr.wikipedia.org/wiki/Composants_web], aka web-components.
+
+Les habitués connaissaient déjà [`angular.directive()`](https://docs.angularjs.org/api/ng/service/$compile), 
+un _helper_ permettant la déclaration de composants réutilisables. 
+`angular.directive()` s'est donc enrichi au cours des années en faisant un atout 
+incontournable du framework de par sa flexibilité et sa simplicité.  
+
+Mais toutes ces possibilités de déclaration n'était plus vraiment en phase avec 
+les spécifications du W3C, `angular.component()` se présente donc un retour aux sources.
```

Comment:
En fait j'explique la raison pour laquelle il forcent le `restrict:'E'`: La [spec de la W3C](http://w3c.github.io/webcomponents/spec/custom/) précise qu'un composant web doit être sous la forme d'un élément nommé `<custom-element></custom-element>` ou `<div is="custom-element"></div>`. La deuxième forme n'existant pas en Angular ils forcent la première. 

Ça te parait aberrant ? Ou alors c'est peut être mal dit. J'vais voir pour reformuler mieux.